### ISSUE TITLE
How to decrease the learning rate while training?

### ISSUE THREAD

**Issue by benni93** [daniel-kukiela/nmt-chatbot#15]
Is the right way to close the terminal, change the parameters in the settings.py and then run train.py again?

**Comment by daniel-kukiela** [comment]
There is something like "better do like that" :) and you have two ways:

1. 
- just break training right after every 1k of steps (when checkpoint is saved)
- change settings in setup/settings.py
- uncomment 'override_loaded_hparams' as well, but only for next training reset
- reset training
but that will reset progress thru training file

2.
- break training process just after next 5k of steps - eval is done every 5k of steps and hparams file inside model folder as well (you wan to keep track of progress thru training set)
- change settings in model/hparams
- reset training
